src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -907,9 +906,35 @@ private static Results runWorkload(List<BenchmarkModule> benchList, boolean verb
         LOG.info("Num New Order transactions : " + numNewOrderTransactions + ", time seconds: " + time);
```

Comment:
Can you please remove `Rate limited req/s` from above and just call it `Throughput`? Also, do you have enough information to calculate throughput per transaction type? (similar to what you're doing for latencies below).

Let's also remove the rate limiting option from the config file - it's confusing for users.

Reply:
Another note: Can we use double for tpmc calculation instead of long and while printing, just use 2 digits after the decimal point?

Reply:
Done.